policy/release/rhtap_multi_ci/rhtap_multi_ci.rego

```diff
@@ -0,0 +1,61 @@
+#
+# METADATA
+# title: RHTAP Multi-CI
+# description: >-
+#   Checks for images built using an RHTAP build pipeline in either Jenkins,
+#   GitLab or GitHub. RHTAP pipelines are defined under
+#   https://github.com/redhat-appstudio/tssc-sample-templates/tree/main/skeleton/ci
+#
+package rhtap_multi_ci
+
+import rego.v1
+
+import data.lib
+
+# METADATA
+# title: SLSA Provenance Attestation Found
+# description: >-
+#   Verify an attestation created by the RHTAP Multi-CI build pipeline is present.
+# custom:
+#   short_name: attestation_found
+#   failure_msg: "A SLSA v1.0 provenance with one of the following RHTAP Multi-CI
+#     build types was not found: %s."
+#   solution: >-
+#     It appears the build pipeline did not create the expected SLSA provenance
+#     attestation. Check for relevant error messages in the 'cosign-sign-attest'
+#     pipeline step logs.
+#   collections:
+#   - rhtap-multi-ci
+#   # For compatibility. These will be removed these soon.
+#   - rhtap-github
+#   - rhtap-gitlab
+#   - rhtap-jenkins
+#
+deny contains result if {
+	count(_attestations) < 1
+	result := lib.result_helper(rego.metadata.chain(), [lib.quoted_values_string(_known_build_types)])
+}
+
+_known_build_types := [_build_type(known_ci_type) | some known_ci_type in _known_ci_types]
+
+_build_type(ci_type) := sprintf("https://redhat.com/rhtap/slsa-build-types/%s-build/v1", [ci_type])
+
+# RHTAP Multi-CI currently supports these environments
+_known_ci_types := ["jenkins", "github", "gitlab"]
```

Comment:
Let's not over complicate this
```suggestion
_known_build_types := [
	"https://redhat.com/rhtap/slsa-build-types/jenkins-build/v1",
	"https://redhat.com/rhtap/slsa-build-types/github-build/v1",
	"https://redhat.com/rhtap/slsa-build-types/gitlab-build/v1",
]
```

Reply:
I like this diff, but I think I wanna use `_known_ci_types` and `_build_type` elsewhere, so it might be weird to not use them when defining _known_build_types. :thinking:

Reply:
I did make a little refactor commit for it, but decided I didn't like it.